Makefile

```diff
@@ -30,14 +39,44 @@ e2e: deploy
 $(TARGETS): vendor/modules.txt
 	./scripts/$@
 
-build-subctl: pkg/subctl/operator/common/embeddedyamls/yamls.go $(shell find pkg/subctl/ -name "*.go")
+clean:
+	rm -f $(BINARIES) $(CROSS_BINARIES) $(CROSS_TARBALLS)
+
+build: operator-image $(BINARIES)
+
+build-cross: $(CROSS_TARBALLS)
```

Comment:
hmmm, and where's the logic for this now?

Reply:
oh, depends on the tarballs.. :)

Reply:
In the `dist/subctl-%.tar.xz` target below.